app/components/SearchTable.tsx

```diff
@@ -13,7 +13,7 @@ class SearchTableComponent extends Component<Props> {
     searchField: null,
     searchValue: null,
     searchDisplay: 'Search by: ',
-    selectedReportingYear: new Date().getFullYear() - 1
+    selectedReportingYear: 2019
```

Comment:
This hardcoding will be problematic once the new reporting period is configured. Added a feature card to fix this properly: https://youtrack.button.is/issue/GGIRCS-2135